suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml

```diff
@@ -154,6 +159,15 @@ tests:
                   sep: ";"            # separator to be used for placements
       destroy-cluster: false
       abort-on-fail: true
+#  - test:
+#      name: Cephadm-ansible client playbook
+#      desc: Configure cephadm client pre-req using cephadm-clients.yml playbook
+#      polarion-id:
+#      module: test_cephadm_ansible.py
+#      config:
+#        playbook: cephadm-clients.yml
+#        extra-vars:
+#          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
```

Comment:
Any reason why it is not included in suite?

Reply:
@Ameenasuhani need some insights here.

Reply:
@sunilkumarn417 are we not executing cephadm-clients playbook here?

Reply:
https://github.com/ceph/cephadm-ansible/blob/devel/cephadm-clients.yml#L1-L24 does this info help?

Reply:
We could either reduce the scope of this PR to remove those tests or have them without commenting it. The pipeline script could be added once the BZ is resolved.

Reply:
working on it. let me check this out once again.
Thanks @psathyan @Ameenasuhani .

Reply:
good that, we found a issue in cephadm-clients playbook.

Reply:
However, code is in and we can enable this as soon as BZ is fixed. https://bugzilla.redhat.com/show_bug.cgi?id=2039413